src/util/sss_sockets.c

```diff
@@ -103,7 +114,7 @@ errno_t set_fd_common_opts(int fd, int timeout)
                   strerror(ret));
```

Comment:
Does `SO_KEEPALIVE` make sense for `AF_UNIX`?

Reply:
It does not, fixed in https://github.com/SSSD/sssd/pull/6484/commits/04b290e4e7335393d91d595057ed22aa7c0459c5